src/commands/lm/lm.ts

```diff
@@ -23,7 +23,7 @@ export const createLmCommand = (program: BaseCommand) => {
 It installs the required credentials helper for Git,
 and configures your Git environment with the right credentials.`,
     )
-    .option('-f, --force', 'Force the credentials helper installation')
```

Comment:
This is another place as well, the -f option no longer works. It is now just `--force` with a generic message